business/checkers/service_checker.go

```diff
@@ -0,0 +1,41 @@
+package checkers
+
+import (
+	"github.com/kiali/kiali/business/checkers/services"
+	"github.com/kiali/kiali/models"
+	v1beta1 "k8s.io/api/apps/v1beta1"
```

Comment:
Oh, nitpicking. I just realized that those imports don't follow the structure. Same thing in the following checker file.